pkg/oc/cli/importimage/importimage.go

```diff
@@ -171,37 +191,45 @@ func (o *ImportImageOptions) Run() error {
 		return err
 	}
 
+	message := "imported"
+	if wasError(result) {
+		message = "imported with errors"
+	}
+
 	if o.DryRun {
-		if wasError(result) {
-			fmt.Fprintf(o.ErrOut, "The dry-run import completed with errors.\n\n")
-		} else {
-			fmt.Fprint(o.Out, "The dry-run import completed successfully.\n\n")
-		}
-	} else {
-		if wasError(result) {
-			fmt.Fprintf(o.ErrOut, "The import completed with errors.\n\n")
-		} else {
-			fmt.Fprint(o.Out, "The import completed successfully.\n\n")
-		}
+		message = fmt.Sprintf("%s (dry run)", message)
 	}
+	message = fmt.Sprintf("%s\n\n", message)
 
 	if result.Status.Import != nil {
+		// TODO: remove once we have external describers
+		var internalResult imageapi.ImageStreamImport
+		if legacyscheme.Scheme.Convert(result, &internalResult, nil); err != nil {
```

Comment:
Don't use `legacyscheme` it's legacy for a reason. Invoke the actual conversion function from image package, like we're doing that already elsewhere (eg. #20343).